src/pyupgrade/settings.rs

```diff
@@ -1,11 +1,14 @@
 //! Settings for the `pyupgrade` plugin.
 
 use ruff_macros::ConfigurationOptions;
+use schemars::JsonSchema;
 use serde::{Deserialize, Serialize};
 
-#[derive(Debug, PartialEq, Eq, Serialize, Deserialize, Default, ConfigurationOptions)]
+#[derive(
+    Debug, PartialEq, Eq, Serialize, Deserialize, Default, ConfigurationOptions, JsonSchema,
+)]
 #[serde(deny_unknown_fields, rename_all = "kebab-case")]
-pub struct Options {
+pub struct PyUpgradeOptions {
```

Comment:
According to the docs, this would also be an option.

```rust
#[serde(deny_unknown_fields, rename_all = "kebab-case", rename = "PyUpgradeOptions")]
pub struct Options {
```